libs/core/src/lib/table/directives/table-cell.directive.ts

```diff
@@ -66,16 +100,82 @@ export class TableCellDirective implements AfterContentInit {
     @HostBinding('class.fd-table__cell')
     _fdTableCellClass = true;
 
-    constructor(public elementRef: ElementRef) {}
+    private readonly _onDestroy$ = new Subject<void>();
+
+    constructor(
+        public readonly elementRef: ElementRef<HTMLElement>,
+        private readonly _liveAnnouncer: LiveAnnouncer,
+        @Optional() private readonly _table?: FdTable
+    ) {}
+
+    ngOnDestroy(): void {
+        this._onDestroy$.next();
```

Comment:
`this._onDestroy$.complete();`.

Reply:
I don't think it's required, is it? Here's the raised question on stackoverflow as an example https://stackoverflow.com/a/44294453/8555571